pkg/appStore/deployment/AppStoreDeploymentService.go

```diff
@@ -93,8 +103,15 @@ func (impl AppStoreDeploymentServiceImpl) AppStoreDeployOperationDB(installAppVe
 		return nil, err
 	}
 
+	var appInstallationMode string
```

Comment:
we can directly assign the variable instead of if else

Reply:
if else is needed due to installAppVersionRequest.AppOfferingMode check in if condition